### ISSUE TITLE
SPC for Recurring payments

### ISSUE THREAD

**Issue by ianbjacobs** [w3c/secure-payment-confirmation#185]
What changes would be needed in SPC to support recurring payment use cases (e.g., display and signing of terms such as recurrence period or number of transactions)? 

It would be good to hear from people whether they think standardized browser UX is appropriate for this situation, or whether the details of recurring payments would make standardization difficult.

**Comment by ianbjacobs** [comment]
cc @stare893

**Comment by SensibleWood** [comment]
Typical details of a recurring payment - using a [standing order](https://openbankinguk.github.io/read-write-api-site3/v3.1.10/resources-and-data-models/pisp/domestic-standing-order-consents.html#obdomesticstandingorder3) from UK open banking as an example - are:

* A first payment amount.
* A final payment amount.
* Date of first payment.
* Frequency (in the reference this is codified according to a pattern set in the OpenAPI document).
* Number of payments.

The UK standing order example also purposefully ignores T&Cs as they are set elsewhere.

In terms of a standardized UX I think this is "doable", with the caveat this would make for a crowded confirmation screen given the amount of information to convey. T&Cs - where not already covered in existing agreements - would most likely need to live elsewhere.

It's also worth noting that "variable recurring payments" has become a thing relatively recently in the UK standards - https://openbankinguk.github.io/read-write-api-site3/v3.1.10/profiles/vrp-profile.html - and as such "could" be initiated using SPC. This, however is more likely to reflect a non-payment authentication experience as is discussed in #186.

**Comment by ianbjacobs** [comment]
Note for future discussion: support both recurrences (with differing parameters potentially varying) and installment payments.

**Comment by gpk00** [comment]
> When issuers will be presented with any of the above mentioned scenarios, they may be forced to not use SPC as currently the SPC UX can not show the user the necessary information about their payment amount, frequency and what amount the users may be authenticating for

I don't think this is a dealbreaker for card transactions and would not consider a blocker nor must have:

1. I don't recall seeing many issuers actually rendering the information made available by merchants on **AReq.recurringFrequency** and **AReq.recurringExpiry** on challenge screens
2. Users will always have the ability to dispute these transactions for non-fraud reasons if they believe the merchant did something wrong.  
3. Also, given the recurring transactions don't actually carry liability shift, users can actually dispute them for fraud reason as well

**Comment by stare893** [comment]
> > When issuers will be presented with any of the above mentioned scenarios, they may be forced to not use SPC as currently the SPC UX can not show the user the necessary information about their payment amount, frequency and what amount the users may be authenticating for
> 
> I don't think this is a dealbreaker for card transactions and would not consider a blocker nor must have:
> 
> 1. I don't recall seeing many issuers actually rendering the information made available by merchants on **AReq.recurringFrequency** and **AReq.recurringExpiry** on challenge screens
> 2. Users will always have the ability to dispute these transactions for non-fraud reasons if they believe the merchant did something wrong.
> 3. Also, given the recurring transactions don't actually carry liability shift, users can actually dispute them for fraud reason as well

The feedback EMVCo has received from European Regulators does paint a landscape where the issuers will be (in time) required to present the the necessary information to the cardholders when they are being authenticated using 3DS. This issue is not about the ability to fight fraud or dispute but making sure the cardholders are fully informed at the time of authentication. The blocker mentioned earlier is related to adoption. With the additional knowledge built regarding recurring transactions in EMV 3DS , merchants and issuers will have more insight into what kinds of transaction is being authenticated and they may lean towards using an authentication method (example issuer's i-frame displaying the break-down of the transaction with OTP/Other authentication method) that gets the cardholders most clarity. The issuers not using the recurring fields today should change over-time with recurring transactions becoming more prevalent and more focus from regulators on these type of transactions.
